docs/config/files.md

```diff
@@ -94,19 +93,19 @@ return {
 and perhaps another one like this:
 
 ```lua
-local wezterm = require 'wezterm';
+local wezterm = require "wezterm"
 return {
-  font = wezterm.font("JetBrains Mono"),
+  font = wezterm.font "JetBrains Mono",
 }
 ```
 
 If you wanted to use both of these in the same file, you would merge them together
 like this:
 
 ```lua
-local wezterm = require 'wezterm';
+local wezterm = require "wezterm"
 return {
-  font = wezterm.font("JetBrains Mono"),
+  font = wezterm.font "JetBrains Mono",
```

Comment:
At the risk of sounding mercurial, this is one case where I would lean towards having parens